src/App.vue

```diff
@@ -59,6 +55,10 @@ export default {
     if (this.girderRest.user) {
       this.fetchFolder();
     }
+    this.girderRest.$on('login', () => {
+      this.girderRest.fetchUser();
```

Comment:
No need to call `fetchUser` here, if you have received the 'login' event, the user will already be set on the `girderRest` instance.